### PR TITLE
Remove circular references from NRF documentation

### PR DESCRIPTION
**Pull request by ppannuto**
The NRF docs pointed you to getting started and getting started pointed you to "board-specific" instructions in these docs.

@niklasad1 could you double-check that I haven't said anything wrong / oversimplified here

### CODE REVIEW

File: boards/nrf51dk/README.md [hunk 1]
```diff
@@ -8,30 +8,27 @@ radio. The kit is Arduino shield compatible and includes several
...
-### Programming with JTAG (recommended)
+### Programming the kernel
 
-The nRF51DK, a Segger JTAG chip is included on the board. Connecting
-the board to your computer over USB allows you to program (and debug)
-Tock with JTAG. To compile and install the Tock kernel on the nrf51dk
-using JTAG, follow the standard Tock instructions (the "Getting
-Started" guide).
+Once you have all software installed, you should be able to simply run
+`make flash` from this directory to install a fresh kernel.
```
Comment (niklasad1) [inline boards/nrf51dk/README.md]: perhaps ```make TOCK_BOARD=nrf51dk flash``` is better here

File: boards/nrf51dk/README.md [hunk 2]
```diff
@@ -8,30 +8,27 @@ radio. The kit is Arduino shield compatible and includes several
...
-## Programming user-level applications
+### Programming user-level applications
 
-To compile and install compile applications for the nrf51dk, follow the
-standard Tock instructions (the "Getting Started" guide).
+After building (`make` in the application folder), you can program an
+application using tockloader:
 
-## Debugging
+    tockloader install --jtag --board nrf51dk --arch cortex-m0 build/blink.tab
```
Comment (niklasad1) [inline boards/nrf51dk/README.md]: *build/blink.tab* indicates that the name of the app is blink right? 
Perhaps ```NAME_OF_THE_APP.tab``` is better?

I haven't used tockloader for this to be honest, you are sure this works?
Tried it briefly and it didn't work perhaps it was some problem with my installation tockloader

Comment (brghena) [inline boards/nrf51dk/README.md]: No, that last bit is actually the path to the tab file. The name is embedded in the tab file itself.

**Comment by ppannuto** [comment]
Updated.

Tockloader is working for me, but please let us know if it isn't working for you for some reason :/
(definitely update it first, it's seen some heavy development/improvements)

**Pull request merged**